react/week3/todo-app/src/App.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import Timer from './Timer';
+import Heading from './Heading';
+import AddTodo from './AddTodo';
+import GetTodo from './GetTodo';
+
+const App = function () {
+
+
+
+
+    return (
+        <div>
+            <Heading />
+            <Timer />
+            <AddTodo />
+            <GetTodo/>
```

Comment:
I don't think this is necessary here, since I see you're using another component to fetch the todos (todoshelper)